kolibri/core/tasks/api.py

```diff
@@ -411,6 +411,28 @@ def startdiskexport(self, request):
 
         return Response(resp)
 
+    @list_route(methods=["post"])
+    def startdataportalsync(self, request):
+        """
+        Initiate a PUSH sync with Kolibri Data Portal.
+
+        """
+        task = {"facility": request.data["facility"], "type": "SYNCDATAPORTAL", "started_by": request.user.pk}
```

Comment:
what would be some other important metadata to include for sync related tasks?

Reply:
This is actually difficult to achieve because calculating the size of the sync happens later in the morango pipeline. It would be useful if there is some callback where we could update the metadata in the task (cc: @rtibbles )

Reply:
For an example of this, see here: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/content/management/commands/deletecontent.py#L113